src/plugin/code.js

```diff
@@ -181,31 +196,104 @@ figma.ui.onmessage = async message => {
     clearRectangleNode();
 
     renderDifferenceRectangle(differences, modifiedFrames);
+
+    focusScreen();
+  }
+
+  if (message.type === "PAGINATION_BUTTON") {
+    const currentSelection = figma.currentPage.selection[0];
+    const differencesLength = differenceRectangleIdList.length;
+
+    if (
+      currentSelection &&
+      differenceRectangleIdList.includes(currentSelection.id)
+    ) {
+      const currentIndex = differenceRectangleIdList.indexOf(
+        currentSelection.id,
+      );
+
+      let nextIndex;
+
+      if (message.content === "prev") {
+        nextIndex =
+          (currentIndex +
+            CONSTANTS.PAGINATION.INDEX_DECREASE +
+            differencesLength) %
+          differencesLength;
+      } else {
+        nextIndex =
+          (currentIndex + CONSTANTS.PAGINATION.INDEX_INCREASE) %
+          differencesLength;
+      }
```

Comment:
P2: else문 없이 사용해도 되지 않을까용..?

```suggestion
        nextIndex =
          (currentIndex + CONSTANTS.PAGINATION.INDEX_INCREASE) %
          differencesLength;
```

Reply:
else 문이 없는 경우 이전버튼 선택시 위의 로직을 한번 실행하고
다음버튼일 경우의 아래의 로직을 한번 더 실행하기 때문에 사용했습니다.